### PR TITLE
Improve Error handling and messaging of longRunning integrations

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37417

## Description
Error handling and error messaging were improved in the 3 following longRunning integrations:
- TAXIIServer
- EDL
- Exportindicators

Bug 37417 reproduced in TAXIIServer but didn't reproduced in EDL and Exportindicators integrations. 
After adding the error handling logic (using demisto.updateModuleHealth(f'An error occurred: {error_message}') to the run_long_running function of the integrations, the bug did reproduced, and then solved by adding the call demisto.updateModuleHealth('') as suggested in the bug issue. 

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
In the EDL Integration - func run_long_running - finally block there are two more error catches. 
I was thinking to add there the error handling logic (using demisto.updateModuleHealth(f'An error occurred: {error_message}')  as well. 
@Itay4 Please let me know if you think it's needed.

**Review by Itay4 — APPROVED** [review]

**Comment by bakatzir** [comment]
@ShacharKidor please bump the relevant pack version and generate rn using: https://xsoar.pan.dev/docs/documentation/release-notes
I'll review them afterwards.

File: Packs/EDL/ReleaseNotes/2_0_4.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS EDL Service
+* Maintenance and stability enhancements.
```
Comment (bakatzir) [inline Packs/EDL/ReleaseNotes/2_0_4.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/ExportIndicators/ReleaseNotes/1_0_8.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Export Indicators Service
+* Maintenance and stability enhancements.
```
Comment (bakatzir) [inline Packs/ExportIndicators/ReleaseNotes/1_0_8.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/TAXIIServer/ReleaseNotes/1_0_7.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### TAXII Server
+* Maintenance and stability enhancements.
```
Comment (bakatzir) [inline Packs/TAXIIServer/ReleaseNotes/1_0_7.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

**Review by bakatzir — APPROVED** [review]

**Pull request merged**